models/RegistrationChecks.php

```diff
@@ -48,6 +48,10 @@ public function rules()
         $module = Yii::$app->getModule('legal');
         $rules = [];
 
+        if (Yii::$app->user->isAdmin()) {
```

Comment:
In this case admin never have to agree or? 
Alternatively maybe something like: Current User is not Registration User or a check based on Scenario?  

In any case we should add here some comments that it is for the case  admin use creation.

Reply:
In this case admin can leave registration checks unchecked and after the first login, the user will have to accept them himself.

Reply:
Yes, that would be ideal, the admin should not check it and the user should have to accept it himself.